tests/moe/test_b12x_fused_moe.py

```
@@ -79,6 +81,99 @@ def _cuda_13_or_newer():
+
+    x = torch.empty((1, 256), dtype=torch.bfloat16)
+    topk_ids = torch.zeros((1, 1), dtype=torch.int32)
+    topk_weights = torch.ones((1, 1), dtype=torch.float32)
+    w1 = torch.empty((2, 384, 128), dtype=torch.uint8)
+    w1_sf = torch.empty((2, 384, 16), dtype=torch.float8_e4m3fn)
+    w2 = torch.empty((2, 256, 96), dtype=torch.uint8)
+    w2_sf = torch.empty((2, 256, 12), dtype=torch.float8_e4m3fn)
+    alpha = torch.ones((2,), dtype=torch.float32)
+    output = torch.empty((1, 256), dtype=torch.bfloat16)
+
+    with pytest.raises(ValueError, match="dynamic.*num_local_experts.*num_experts"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)